web/themes/custom/cloudy/templates/content/service-location/node--service-location--related.html.twig

```diff
@@ -25,20 +25,26 @@
       {% if node.field_location[0].entity.field_address_or_entrance.locality %}
         {{ node.field_location[0].entity.field_address_or_entrance.locality }}{% if node.field_location[0].entity.field_address_or_entrance.administrative_area %}, {{ node.field_location[0].entity.field_address_or_entrance.administrative_area }}{% endif %}{% if node.field_location[0].entity.field_address_or_entrance.postal_code %} {{ node.field_location[0].entity.field_address_or_entrance.postal_code }}{% endif %}
       {% endif %}
-  </address>
-{% endspaceless %}
-{% if node.field_emergency_closure.value == 1 %}
-  <div class="location--closure">
-    <span class="badge badge-danger">Emergency Closure</span><br>
-    <a href="/emergency-closures">Emergency closure information</a>
-  </div>
-{% else %}
-  {% if node.field_location[0].entity.field_geolocation is empty %}
-    <div class="get-directions-link"><a href="https://www.google.com/maps/dir/{{ node.field_location[0].entity.field_address_or_entrance.address_line1 }} {{ node.field_location[0].entity.field_address_or_entrance.address_line2 }} {{ node.field_location[0].entity.field_address_or_entrance.locality }} {{ node.field_location[0].entity.field_address_or_entrance.administrative_area }} {{ node.field_location[0].entity.field_address_or_entrance.postal_code }}" target="_blank">Get Directions</a></div>
+    </address>
+  {% endspaceless %}
+  {% if node.field_emergency_closure.value == 1 %}
+    <div class="location--closure">
+      <span class="badge badge-danger">Emergency Closure</span><br>
+      <a href="/emergency-closures">Emergency closure information</a>
+    </div>
   {% else %}
-    <div class="get-directions-link"><a href="https://www.google.com/maps/dir/{{ node.field_location[0].entity.field_geolocation.lat }},{{ node.field_location[0].entity.field_geolocation.lon }}" target="_blank">Get Directions</a></div>
+    {% if node.field_location[0].entity.field_geolocation is empty %}
+      <div class="get-directions-link">
+        <a href="https://www.google.com/maps/dir/{{ node.field_location[0].entity.field_address_or_entrance.address_line1 }} {{ node.field_location[0].entity.field_address_or_entrance.address_line2 }} {{ node.field_location[0].entity.field_address_or_entrance.locality }} {{ node.field_location[0].entity.field_address_or_entrance.administrative_area }} {{ node.field_location[0].entity.field_address_or_entrance.postal_code }}" target="_blank">Get Directions</a>
+      </div>
+    {% else %}
+      <div class="get-directions-link">
+        <a href="https://www.google.com/maps/dir/{{ node.field_location[0].entity.field_geolocation.lat }},{{ node.field_location[0].entity.field_geolocation.lon }}" target="_blank">Get Directions</a>
+      </div>
+    {% endif %}
+    {{ content|without('field_location','field_floor','field_emergency_closure') }}
+    <div class="more-information">
+      <a href="{{ url }}" rel="bookmark">More about this location</a>
```

Comment:
"More about this location" isn't being displayed. We should also translate the string.

Reply:
It's not intended to display if a location has the emergency closure triggered. Otherwise this works.

Reply:
Why do we hide it if there's an emergency closure? https://powr-2755-portlandor.pantheonsite.io/service-locations/development-services-center "More about this location" is displayed.